### ISSUE TITLE
Players Unable to Connect to Server: Internal Exception. 

### ISSUE THREAD

**Issue by copbot10011** [xJon/The-1.12.2-Pack#835]
Multiple players are unable to connect to a server and receive the following error client-side when attempting to log in. https://imgur.com/a/70ZRvnd

The issue appears to be a Java Internal Exception for the Minecraft: Crafting Table
 The server has been restarted with no change. Players cannot access the server over WAN or LAN.
 Since the issue is with a vanilla Minecraft item, mod isolation testing cannot be attempted. 
 Server logs: https://mclo.gs/y0QSl8R

**Comment by copbot10011** [comment]
As suggested by a contributor on the 1.12.2 Discord. I am going to attempt to overwrite the FastWorkbench configs.

**Comment by copbot10011** [comment]
Unable to locate config files for FastWorkbench

**Comment by copbot10011** [comment]
fastbench.cfg is not in the ZIP file obtained from https://the-1122-pack.com/repo

**Comment by copbot10011** [comment]
fastbench.cfg was imported from a previous known-good 1.5.1 server config as the CFG file cannot be located in the ZIP (probably needs to be extracted and ran in order to appear). The client attempted to connect to the server using a newly installed client and was still unable to connect getting the same error. 

Requesting assistance in diagnosing this issue.

**Comment by copbot10011** [comment]
The issue started yesterday. No attempts have been made to rollback a world as this doesn’t appear to be a world issue at this time.

**Comment by xJon** [comment]
Have you tried to completely reinstall the modpack? This issue can often be caused due to file corruption.

**Comment by copbot10011** [comment]
Yes. Another player did. There were no updates or configuration changes. The fact this impacts two separate players at the same time points to a server issue unless a latch was pushed out for the modpack less than 48 hours ago.

**Comment by xJon** [comment]
Can you regard this part of my message?
> Edit: Can you also first try to add [LocaleFixer](https://www.curseforge.com/minecraft/mc-mods/localefixer) to the server and client and let me know whether that resolves the issue?

**Comment by copbot10011** [comment]
Lines 2215 and 2216 in the attached server logs appear to be the issue

**Comment by copbot10011** [comment]
I can attempt a test of locale fixer later today. Does this need to be installed on the client, server or both?

**Comment by xJon** [comment]
I'd appreciate if you can test all of the options.

**Comment by copbot10011** [comment]
The issue was able to be resolved by generating new server files from the latest 1.12.2 pack server download.
World and player files were copied over to the new server.
